libusb/libusbi.h

```diff
@@ -1427,6 +1427,14 @@ struct usbi_os_backend {
 	 */
 	int (*handle_transfer_completion)(struct usbi_transfer *itransfer);
 
+	/* Return information about the backend driver. This string contain
+	* at least the backend's driver name. Returns empty string is no info
+	* available or not implemented.
+	*
+	* Providing an implementation for this method is optional
+	*/
+	const char* (*get_driver_name)(void);
```

Comment:
if I didn't miss anything - it is always a constant, i.e. doesn't require any runtime evaluation to retrive the name of the "driver",
so it might be just a variable/member (same way as `name`) instead of a function

Reply:
On Windows backend, there can be two drivers, `usbk` or `WinUSB`. These are dynamically picked from `windows_init` and also dynamically changed via `windows_set_option`.

Reply:
right, I did miss this one